bench_lib.rb

```diff
@@ -79,20 +119,28 @@ def server_cleanup
         return csystem(@server_kill_command, "Failure when running server kill command!", fail_ok: true)
       end
       pids = running_server_pids
-      return if pids == []
-      pids.each { |pid| Process.kill "HUP", pid }
-      sleep 3 # Leave time to clean up after SIGHUP
-      pids = running_server_pids
-      pids.each { |pid| Process.kill "KILL", pid }
+      return if pids.empty?
+      pids.each { |pid| Process.kill "INT", pid }
```

Comment:
To make sure I understand... This will *never* send a SIGKILL, correct? So if the process never responds, this will wait forever, or until manually canceled.

Reply:
Right, the current logic will not.
If you think it's an issue, we could probably add a timeout around `Process.wait(pid)` and if that takes more than 1-3 second, then SIGKILL and wait again. Should I do that?

Reply:
I think that'd be an improvement, yeah.

Reply:
Here we go: https://github.com/noahgibbs/rsb/pull/3